README.md

```diff
@@ -293,6 +293,29 @@ To update the left navigation after changing an endpoint's path follow the steps
 4. Send the PR in `#dev-portal-pr` Slack channel for approval.
 5. After approval, merge the PR.
 
+### How to remove an API reference from Developer Portal?
+
+You will need to create two pull requests in the following order: one for `devportal` repository and other for `openapi-schemas`.
+
+To remove the API content from `devportal` repository, follow the steps below:
+
+1. Open a branch in the [devportal](https://github.com/vtexdocs/devportal) repository.
+2. In the `src/pages/api/openapi/[slug].tsx` file, remove the content that refers to the API. It follows this syntax: `'{schemaFileName}': '{slug}'`.
+3. In the `src/utils/getReferencePaths.ts` file, remove the content that refers to the API. It follows this syntax: `'{schemaFileName}': '{slug}'`.
+4. Open a PR with the changes.
```

Comment:
I suggest adding a step to remove the API and its endpoints from navigation.json

Reply:
Maybe it's worth adding the redirect as an optional step here instead of in a callout in the end, since it's in the same repository

Reply:
Agree and changed it!